src/org/ods/orchestration/service/JiraService.groovy

```diff
@@ -793,4 +800,66 @@ class JiraService {
         }
         return new JsonSlurperClassic().parseText(response.getBody())
     }
+
+    def getIssueStatusWithTransitions(issueId) {
```

Comment:
Convert this method in a generic method "getIssue" with an optional param 'expand' (the string that is going to be passed to rest API) and returning the whole map.

Reply:
Done.